### ISSUE TITLE
should not shutdown when cassandra server is down

### ISSUE THREAD

**Issue by fmarslan** [avalanche123/cassandra-web#18]
the app should not shutdown when Cassandra server is down. is there a solution for this ?

**Comment by avalanche123** [comment]
do you mean that you can't start it when the server is down? you can keep starting it continuously until it succeeds or contribute re-initializing a cluster object

**Comment by fmarslan** [comment]
Yes, I think if the app not stops and tries to connect instead, will be better. Can this solution applicable to software?
